R/formatters_var_labels.R

```diff
@@ -25,7 +25,7 @@
 #' col_labels(x) <- paste("label for", names(iris))
 #' col_labels(x)
 col_labels <- function(x, fill = FALSE) {
-  stopifnot(is.data.frame(x))
+  checkmate::assert_character(colnames(x), null.ok = FALSE, any.missing = FALSE)
```

Comment:
Why not `assert_data_frame`?
Also, `null.ok = FALSE` is the default.

Reply:
> Why not assert_data_frame?

x can be of  class  `data.frame` or `DataFrame`.